generators/app/templates/_material/sources/translations/_en-US.po

```diff
@@ -0,0 +1,34 @@
+msgid ""
```

Comment:
an update for en/fr translations should be made based on `gulp translations:extract`